OPHD/States/MapViewStateHelper.cpp

```diff
@@ -313,29 +313,31 @@ bool selfSustained(StructureID id)
 /** 
  * Indicates that a specified tile is out of communications range (out of range of a CC or Comm Tower).
  */
-bool outOfCommRange(Point<int>& cc_location, TileMap* tile_map, Tile* current_tile)
+bool inCommRange(NAS2D::Point<int> position)
 {
-	Tile* tile = tile_map->getVisibleTile();
-
-	if (tile->distanceTo(tile_map->getTile(cc_location, 0)) <= constants::ROBOT_COM_RANGE)
-		return false;
-
-	Tile* _comm_t = nullptr;
-	for (auto _tower : Utility<StructureManager>::get().structureList(Structure::StructureClass::CLASS_COMM))
+	const auto maxCcRangeSquared = constants::ROBOT_COM_RANGE * constants::ROBOT_COM_RANGE;
+	const auto ccDistance = position - ccLocation();
+	if (ccDistance.lengthSquared() <= maxCcRangeSquared)
+		return true;
```

Comment:
I think we are standardizing to using braces on one line conditional logic statements

Reply:
Ahh, right, I noticed that and then forgot.